services/ui-src/src/components/report/ReportAutosaveProvider.test.tsx

```diff
@@ -49,6 +51,8 @@ describe("<UserProvider />", () => {
   test("test autosave function", async () => {
     const saveButton = screen.getByRole("button", { name: "Save" });
     await userEvent.click(saveButton);
-    await waitFor(() => expect(mockSaveReport).toHaveBeenCalled);
+    await waitFor(() => expect(mockSaveReport).toHaveBeenCalled(), {
+      timeout: 3000,
+    });
```

Comment:
Isn't the default timeout sufficient here?

```suggestion
    await waitFor(() => expect(mockSaveReport).toHaveBeenCalled());
```